src/univ/textctrl.cpp

```diff
@@ -2459,7 +2454,7 @@ void wxTextCtrl::UpdateLastVisible()
         return;
 
     // use (efficient) HitTestLine to find the last visible character
-    wxString text = m_value.Mid((size_t)SData().m_colStart /* to the end */);
+    wxString text = m_value;//m_value.Mid((size_t)SData().m_colStart /* to the end */);
```

Comment:
Very minor, but let's not keep comments out text:
```suggestion
    wxString text = m_value;
```

Also, do we actually need this variable now, i.e. couldn't we just use `m_value` instead?

Reply:
Yes, because it's modify own state by Truncate. Comment removed.